starfyre/__main__.py

```diff
@@ -42,16 +42,26 @@ def create_main_file(path):
 
 @click.command()
 @click.option("--path", default=".", help="Path to the project")
-@click.option("--dev", default=False, help="Start the compilation and generate the build package.")
-@click.option("--build", default=False, help="Start the build package")
+@click.option("--dev/--no-dev", default=False, help="Start the compilation and generate the build package.")
+@click.option("--build/--no-build", default=False, help="Start the build package")
```

Comment:
What is the purpose of having a --no-dev or a --no-build flag here?? It makes the interface a bit more confusing imo.

What we are looking for is the is_flag argument. You can read more about it here(https://pymbook.readthedocs.io/en/latest/click.html)

Reply:
Thanks for this! Updated.